test/scripts/test_gas_diff_stats.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python
+
+import unittest
+from textwrap import dedent
+
+# NOTE: This test file file only works with scripts/ added to PYTHONPATH so pylint can't find the imports
+# pragma pylint: disable=import-error
```

Comment:
Shouldn't we add it then? I.e.:
```python
import sys
from pathlib import Path
PROJECT_ROOT = Path(__file__).parents[2]
sys.path.insert(0, f"{PROJECT_ROOT}/scripts")
```

Reply:
We do, but not here. In the test runner: [`test/pyscriptTests.py`](https://github.com/ethereum/solidity/blob/develop/test/pyscriptTests.py)

Reply:
Oh, I missed that `pyscriptTests.py`. Good to know! ;)